js/main.js

```diff
@@ -22,16 +22,30 @@ function handleSubmit(event) {
     photoURL: $entryForm[1].value,
     notes: $entryForm[2].value,
   };
-  inputValues.entryId = data.nextEntryId;
-  data.nextEntryId++;
-  data.entries.unshift(inputValues);
-  $image.setAttribute('src', 'images/placeholder-image-square.jpg');
-  $entryForm.reset();
-  $ul.prepend(renderEntry(inputValues));
-  viewSwap('entries');
-  if (data.entries !== null) {
-    toggleNoEntries();
+  if (data.editing === null) {
+    inputValues.entryId = data.nextEntryId;
+    data.nextEntryId++;
+    data.entries.unshift(inputValues);
+    $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+    $ul.prepend(renderEntry(inputValues));
+    if (data.entries !== null) {
+      toggleNoEntries();
+    }
+  } else if (data.editing !== null) {
```

Comment:
This syntax is incorrect, you have 2 closing braces after each other. It should be `} else if`. You can also just do `else` rather than `else if`, because we've already verified if they were editing or not